Dockerfile

```diff
@@ -193,83 +279,27 @@ RUN R -e "remotes::install_github('jokergoo/circlize', ref = 'b7d86409d7f893e881
 # signature.tools.lib needed for mutational-signatures 
 RUN R -e "remotes::install_github('Nik-Zainal-Group/signature.tools.lib', ref = 'a54e5d904d091b90ad3b0f9663133e178c36b9aa', dependencies = TRUE)"
 
-# Install python packages
-##########################
+# Molecular subtyping MB
+RUN R -e "remotes::install_github('d3b-center/medullo-classifier-package', ref = 'e3d12f64e2e4e00f5ea884f3353eb8c4b612abe8', dependencies = TRUE, upgrade = FALSE)"  
+    
+# More recent version of sva required for molecular subtyping MB
+RUN R -e "remotes::install_github('jtleek/sva-devel@123be9b2b9fd7c7cd495fab7d7d901767964ce9e', dependencies = FALSE, upgrade = FALSE)"
 
-# Install python3 tools and ALL dependencies
-RUN pip3 install \
-    "appdirs==1.4.4" \
-    "attrs==23.1.0" \
-    "certifi==2023.5.7" \
-    "chardet==5.1.0" \
-    "ConfigArgParse==1.5.3" \
-    "CrossMap==0.6.5" \
-    "defusedxml==0.7.1" \
-    "docutils==0.20" \
-    "gitdb==4.0.10" \
-    "GitPython==3.1.31" \
-    "idna==3.4" \
-    "importlib-metadata==6.6.0" \
-    "ipykernel==6.23.0" \
-    "ipython==8.13.2" \
-    "ipython-genutils==0.2.0" \
-    "jsonschema==4.17.3" \
-    "jupyter-client==8.2.0" \
-    "jupyter-core==5.3.0" \
-    "MarkupSafe==2.1.2" \
-    "matplotlib==3.7.1" \
-    "nbconvert==7.4.0" \
-    "nbformat==5.8.0" \
-    "notebook==6.5.4" \
-    "numpy==1.24.3" \
-    "packaging==23.1" \
-    "palettable==3.3.3" \
-    "pandas==2.0.1" \
-    "pandocfilters==1.5.0" \
-    "parso==0.8.3" \
-    "patsy==0.5.3" \
-    "pexpect==4.8.0" \
-    "pickleshare==0.7.5" \
-    "plotnine==0.12.1" \
-    "prompt-toolkit==3.0.38" \
-    "psutil==5.9.5" \
-    "ptyprocess==0.7.0" \
-    "Pygments==2.15.1" \
-    "pyparsing==3.0.9" \
-    "python-dateutil==2.8.2" \
-    "pytz==2023.3" \
-    "PyYAML==6.0" \
-    "pyzmq==25.0.2" \
-    "ratelimiter==1.2.0.post0" \
-    "requests==2.30.0" \
-    "rpy2==3.5.0" \
-    "scikit-learn==1.2.2" \
-    "scipy==1.10.1" \
-    "seaborn==0.12.2" \
-    "six==1.16.0" \
-    "snakemake==7.25.3" \
-    "statsmodels==0.14.0" \
-    "tornado==6.3.1" \
-    "traitlets==5.9.0" \
-    "urllib3==2.0.2" \
-    "utils==1.0.1" \
-    "webencodings==0.5.1" \
-    "widgetsnbextension==4.0.7" \
-    "wrapt==1.15.0" \
-    "zipp==3.15.0" \
-    "openpyxl==3.1.2" \
-    && rm -rf /root/.cache/pip/wheels
+# Packages required for de novo mutational signatures
+#RUN install2.r --error --deps TRUE \
+#    lsa
```

Comment:
Were these two lines meant to be commented out?

Reply:
Yes, not using this package at the moment